karma.conf.maker.js

```diff
@@ -92,10 +92,10 @@ function setBrowsers(karmaConf, browserstack) {
   }
 }
 
-module.exports = function(codeCoverage, browserstack, watchMode) {
+module.exports = function(codeCoverage, browserstack, watchMode, file) {
   var webpackConfig = newWebpackConfig(codeCoverage);
   var plugins = newPluginsArray(browserstack);
-  var files = [
+  var files = file ? [file] : [
```

Comment:
I think this needs to be `var files = file ? ['test/helpers/prebidGlobal.js', file] : ...`

Otherwise it fails on files which use `$$PREBID_GLOBAL$$`--even if it's just indirectly, through the import chain. For example, `gulp test --file "test/spec/modules/appnexusAstBidAdapter_spec.js"` fails because the adapter relies on `src/config`, which relies on `$$PREBID_GLOBAL$$`.

This problem should fade away as progress gets made on #1508 and #1510... but for now it's common enough that the gulpfile should probably support it.